tensorboard/webapp/metrics/views/card_renderer/histogram_card_test.ts

```diff
@@ -276,6 +278,38 @@ describe('histogram card', () => {
   });
 
   describe('linked time', () => {
+    it('dispatches linkedTimeSelectionChanged when HistogramComponent emits onLinkedTimeSelectionChanged event', () => {
+      provideMockCardSeriesData(selectSpy, PluginType.HISTOGRAMS, 'card1');
+      store.overrideSelector(selectors.getMetricsLinkedTimeSelection, {
+        start: {step: 5},
+        end: null,
+      });
+      const fixture = createHistogramCardContainer();
+      fixture.detectChanges();
+      const dispatchedActions: Action[] = [];
+      spyOn(store, 'dispatch').and.callFake((action: Action) => {
+        dispatchedActions.push(action);
+      });
+
+      const HistogramWidget = fixture.debugElement.query(
```

Comment:
style nit: HistogramWidget => histogramWidget

Reply:
hmm not sure why it was capital H for other previous tests. Updated them all.